crates/topos/src/components/subnet/commands.rs

```diff
@@ -0,0 +1,29 @@
+use clap::{Args, Subcommand};
+
+mod subnet;
+
+pub(crate) use subnet::Run;
+
+#[derive(Args, Debug)]
+pub(crate) struct SubnetCommand {
```

Comment:
Just wanted to mention that doc-comments would be nice here as well (same as with the `Run` struct). Don't know if we want to be this picky about it, so just wanted to raise my hand :)